cloud/logger-cloud/src/Logger.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Created by acunningham on 15/06/17.
+ */
+
+
+
+export interface Logger {
+    //log(message:string, object:any): void;
+    debug(message:string, object:any): void;
+    error(message:string, object:any): void;
+    info(message:string, object:any): void;
+    warn(message:string, object:any): void;
+}
+
+export enum LOG_LEVEL {
```

Comment:
I think this is valid use case for this https://github.com/witmicko/raincatcher-core/blob/7f4ce88a5ce6313c1cfb4ce283dbc2ed1e60b3fd/cloud/logger-debug/src/index.ts#L46